systemtests/aci_test.go

```diff
@@ -26,45 +26,38 @@ func (s *systemtestSuite) TestACIMode(c *C) {
 		Encap:       "vlan",
 	}), IsNil)
 
-	err := s.nodes[0].checkDockerNetworkCreated("aciNet", false)
+	err := s.nodes[0].checkSchedulerNetworkCreated("aciNet", false)
 	c.Assert(err, IsNil)
 
 	c.Assert(s.cli.EndpointGroupPost(&client.EndpointGroup{
 		TenantName:  "default",
 		NetworkName: "aciNet",
-		GroupName:   "epgA",
+		GroupName:   "epga",
```

Comment:
kubernetes doesnt accept pod names to be non dns names. Since we use epg names as pod/container names. i changed the epg name